doc/src/site/sphinx/install/install_and_start.rst

```diff
@@ -0,0 +1,205 @@
+Installing and Starting
+=======================
+
+This section describes how to download and run Sparkling Water in different environments. Refer to the :ref:`pysparkling` and :ref:`rsparkling` sections for instructions on installing and running PySparkling and RSparkling. 
+
+Download and Run Locally
+------------------------
+
+This section describes how to quickly get started with Sparkling Water on your personal computer (in Spark's ``local`` cluster mode).
+
+1. Download and install Spark (if not already installed) from the `Spark Downloads page <https://spark.apache.org/downloads.html>`__.
+
+  ::
+
+    Choose Spark release: SUBST_SPARK_VERSION
+    Choose package type: Pre-built for Hadoop 2.7 and later
+
+2. Point SPARK_HOME to the existing installation of Spark and export variable MASTER.
+
+  ::
```

Comment:
This should be explicitly marked as bash code as `    .. code:: bash` to be consistent with the rest of the doc. Applies to all `::`

Reply:
Oh, that's right. And SW docs don't include the osexample extension, so I can't do tabbed examples.